### ISSUE TITLE
Logger middleware: constant value cannot be serialized

### ISSUE THREAD

**Issue by andreasrueedlinger** [inversify/InversifyJS#391]
## Expected Behavior

Using a constant value binding should not break the logger middleware and output a corrsponding serialization.
## Current Behavior

It fails with the error `Cannot read property 'name' of null` for `implementationDetail
## Possible Solution

Not only check for `undefined` but also for `null` values
## Steps to Reproduce (for bugs)

User logger middleware, bind a constant value eg. `kernel.bind<string>('string').toConstantValue('foo')` and use it in a inject.
# Stack trace

```
***/node_modules/inversify-logger-
middleware/lib/serializers/text/binding_serializer.js:26                    
                    val = _b[prop].name;                                    
                                  ^                                         

TypeError: Cannot read property 'name' of null                              
    at ***/node_modules/inversify-
logger-middleware/lib/serializers/text/binding_serializer.js:26:35          
    at Array.forEach (native)                                               
    at Object.serializeBinding [as default]
```

**Comment by remojansen** [comment]
Thanks for reporting this issue. We will fix it ASAP feel free to send a PR if you want to contribute a fix (Please remember to add a test case to reproduce the issue before fixing).

**Comment by remojansen** [comment]
Done by https://github.com/inversify/inversify-logger-middleware/pull/32 & released by with `inversify-logger-middleware@2.0.3` 🎉
